### PR TITLE
Merge dev to prod

### PR DESCRIPTION
**Pull request by sowmyaraj92**

### CODE REVIEW

**Review by shashankawasthi88 — APPROVED** [review]
Looks good.

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 1]
```diff
@@ -95,6 +93,137 @@ The broader development community has published some useful tutorials for settin
 * [macOS](https://support.apple.com/en-gb/guide/mac-help/mchlp2591/mac)
 * [Android](https://www.howtogeek.com/295048/how-to-configure-a-proxy-server-on-android/)
 
-## Interceptor
```
[the remark this answers is not in the document]
Comment (sowmyaraj92) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:98]: @shashankawasthi88 As discussed, we will make this change and push it to prod in the next release.
@jerry-li-dev Please do raise a PR for this change, lets get it done ASAP.

[the remark this answers is not in the document]
Comment (jerry-li-dev) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:98]: PR here: https://github.com/postmanlabs/postman-docs/pull/2969

[the remark this answers is not in the document]
Comment (sowmyaraj92) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:98]: @jerry-li-dev Please make sure the images are sized properly with a border of 1px.

[the remark this answers is not in the document]
Comment (jerry-li-dev) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:98]: All images converted to jpg and a has a boarder. Please approve so we can push this PR to prod.

**Review by brandonmcastillo — APPROVED** [review]
LGTM for prod

**Pull request merged**